### PR TITLE
Fix "Click Action From Actions Menu" and CustomServingRuntime test

### PR DESCRIPTION
**Pull request by bdattoma**
Fixing `Verify RHODS Admins Can Delete A Custom Serving Runtime Template` which got broken after patternfly update. 
It involves fixing `Click Action From Actions Menu` kw
- making the xpath slightly more robust, hopefully
- replace keyword duplications in Workbenches, DataConnections,Storage and Projects resources in order to reduce the maintanance (I didn't change it in Pipelines resource files because I'm unable to test the change due to some infrastructure issues. Will do in a later PR)

### CODE REVIEW

**Review by diegolovison — APPROVED** [review]

**Comment by bdattoma** [comment]
**PR Validation**
1. CustomServingRuntime suite: `rhods-ci-pr-test/2312` Running
2. Smoke suite: coming soon..

**Review by FedeAlonso — APPROVED** [review]
LGTM

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot [hunk 1]
```diff
@@ -837,9 +837,9 @@ Handle Deletion Confirmation Modal
 Click Action From Actions Menu
     [Documentation]    Clicks an action from Actions menu (3-dots menu on the right)
     [Arguments]    ${item_title}    ${action}    ${item_type}=${NONE}
-    Click Element       xpath=//tr[td[@data-label="Name"]//*[text()="${item_title}"]]/td[@class="pf-v5-c-table__action"]/div/button[@aria-label="Actions"]
+    Click Element       xpath=//tr[td[@data-label="Name"]//*[text()="${item_title}"]]/td[contains(@class,"-table__action")]//button[@aria-label="Kebab toggle"]    # robocop: disable
```
Comment (jiridanek) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:840]: Note to self, for the future we should have keywords for clicking common paternfly UI elements. Locators of this style appear many times. `@aria-label="Kebab toggle"` is beginning to hound my dreams.

**Review by jiridanek — APPROVED** [review]

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot (hunk 1 above)
Comment (bdattoma) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:840]: yep I agree, that would help also reducing the code duplication. However I don't kow now from top of my head some more keywords like this one we could have. But if you find any, welcome to have it in a common place to be re-used

**Pull request merged**